pkg/chains/address.go

```diff
@@ -54,6 +54,7 @@ func ConvertRecoverToError(r interface{}) error {
 
 // DecodeBtcAddress decodes a BTC address from a given string and chainID
 func DecodeBtcAddress(inputAddress string, chainID int64) (address btcutil.Address, err error) {
+	// prevent potential panic from 'btcutil.DecodeAddress'
```

Comment:
nit: not in the scope of this PR, but is this defer recover really needed? Going through `DecodeAddress` code shows that there are no panic raised by it or functions called from it.

Reply:
Context: a panic was thrown in the past, so this defer was then introduced as a guard.